signals/content/subject_is_upper.yml

```
@@ -0,0 +1,4 @@
+name: "Subject: Contains All Capital Letters"
+type: "query"
+source: |
+  regex.match(subject.subject, "^[A-Z]*$")
```

```suggestion
  regex.match(subject.subject, "^[A-Z ]*$")
```

I think you might want a space in your char set? Otherwise you'll not be able to match things like "TEST SUBJECT"?